### PR TITLE
Add setup.py for automated testing

### PR DESCRIPTION
**Pull request by dagwieers**
Using a setup.py file in your project directory makes it possible for other projects to pull in this library directly from Git (e.g. from requirements.txt in a CI environment).

We need this for testing our add-on using Addon Signals in Travis CI during integration testing.

After this is merged, one can add the repository like this to *requirements.txt*:
```
git+git://github.com/ruuk/script.module.addon.signals.git@master#egg=AddonSignals
```

And then install all dependencies for CI (e.g. in .travis.yml) using:
```bash
pip install -r requirements.txt
```

### CODE REVIEW

**Comment by dagwieers** [comment]
*(Please hold off merging this yet, since it's a single-file package it needs some work)*

**Comment by dagwieers** [comment]
This is ready to be merged.

File: addon.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 		<summary lang="en">Inter-addon signalling</summary>
 		<description lang="en">Provides signal/slot mechanism for inter-addon communication</description>
 		<platform>all</platform>
-		<license>GNU GENERAL PUBLIC LICENSE Version 2.1, February 1999</license>
+		<license>LGPL-2.1-only</license>
```
Comment (ruuk) [inline addon.xml:11]: While I'm not necessarily opposed to this change, I'm curious as to the reason for it.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Dag Wieers (@dagwieers) <dag@wieers.com>
```
Comment (ruuk) [inline setup.py]: The copyright line seems out of place here.

File: addon.xml (hunk 1 above)
Comment (dagwieers) [reply addon.xml:11]: It is required in the future. In reviews of my add-ons I was asked to change this.

File: setup.py (hunk 2 above)
Comment (dagwieers) [reply setup.py]: I will remove it, in other projects we use copyright statements in every file as is legally required.
In this case I would say *setup.py* is even not copyrightable.

File: addon.xml (hunk 1 above)
Comment (dagwieers) [reply addon.xml:11]: https://github.com/xbmc/repo-scripts/pull/1213#pullrequestreview-321911606

**Review by ruuk — APPROVED** [review]
LGTM!

**Pull request merged**